packages/core/src/code-evaluation/serializeSolids.js

```diff
@@ -14,7 +14,7 @@ const serializeSolids = (solids) => {
     if (geom2.isA(object)) geom2.toSides(object)
     if (geom3.isA(object)) geom3.toPolygons(object)
     if (path2.isA(object)) path2.toPoints(object)
-    return JSON.stringify(object)
+    return object
```

Comment:
Let's add another NOTE... something like...

NOTE: Serialization of objects is the fastest due to optimizations in the JavaScript engines.

Reply:
I have adeed a note. Is this ok now  for merging ? :)